### ISSUE TITLE
predicted HOG target visualization

### ISSUE THREAD

**Issue by ghost** [ggjy/FastMIM.pytorch#2]
Hey, I am grateful for your excellent work .  As mentioned in your paper and maskfeat, do you know how we could visualize the predicted HOG target? During model training, I set the parameter gaussian_window in the HOGLayerC class to 16  and args.norm_ fix_ loss to false. By reshaping the HOG target[B,64,108], I can get the general outline of HOG target feature, but it is still different from the one in the paper.
